incubator/couchdb/README.md

```diff
@@ -0,0 +1,93 @@
+# CouchDB
+
+Apache CouchDB is a database featuring seamless multi-master sync, that scales
+from big data to mobile, with an intuitive HTTP/JSON API and designed for
+reliability.
+
+This chart deploys a CouchDB cluster as a StatefulSet. It creates a ClusterIP
+Service in front of the deployment for load balancing by default, but can also
+be configured to deploy other Service types or an Ingress controller. The
+default persistence mechanism is simply the ephemeral local filesystem, but
+production deployments should set `persistentVolume.enabled` to `true` to attach
+storage volumes to each Pod in the deployment.
+
+## TL;DR
+
+```bash
+$ helm install incubator/couchdb --set allowAdminParty=true
+```
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+First, create a Secret containing `adminUsername`, `adminPassword` and `cookieAuthSecret` keys:
+
+```bash
+$ kubectl create secret generic my-release-couchdb --from-literal=adminUsername=foo --from-literal=adminPassword=bar --from-literal=cookieAuthSecret=baz
```

Comment:
you might need to add a test to generate these for CI to function properly

Reply:
Thanks for taking a look! Not sure exactly what you mean by adding a test here. Isee one pattern of programmatically generating default Secrets and allowing an override from yalues.yaml, e.g.

https://github.com/kubernetes/charts/blob/master/stable/mongodb/templates/secrets.yaml

Is that what you'd recommend?

Reply:
OK this is done in 766c84c